models/src/main/thrift/content/v1.thrift

```diff
@@ -882,6 +884,18 @@ struct Tag {
     17: optional string twitterHandle
 }
 
+struct QuizAtom {
+  1: required string id
+  2: required list<string> labels // required, but may be empty
+  3: required string defaultHtml
+  4: required quiz.QuizAtom data       // the atom payload
+//  6: required ContentChangeDetails contentChangeDetails
+ }
+
+/* an embedded content item with an independant lifecycle */
+struct Atoms {
+   1: optional QuizAtom quiz
+}
```

Comment:
So should the Atom model in contentatom.thrift replace Quiz Atom, so that line would look like this: 

```thrift
17: optional contentatom.Atom atoms
```